docs/api/en/helpers/GridHelper.html

```diff
@@ -32,8 +32,9 @@ <h3>[name]( [param:number size], [param:Number divisions], [param:Color colorCen
 		<p>
 		size -- The size of the grid. Default is 10. <br />
 		divisions -- The number of divisions across the grid. Default is 10. <br />
-		colorCenterLine -- The color of the centerline. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x444444 <br />
-		colorGrid -- The color of the lines of the grid. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x888888
+		colorXCenterLine -- The color of the X-centerline. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x444444 <br />
+		colorGrid -- The color of the lines of the grid. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x888888 <br />
+		colorZCenterLine -- The color of the Z-centerline. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x444444
```

Comment:
Unfortunately, the parameter order is now confusing. It should be at least: `colorXCenterLine`, `colorZCenterLine`, `colorGrid`. Or even `colorGrid`,  `colorXCenterLine`, `colorZCenterLine`. However, this will break backwards compatibility. 

In any event, I think we can't accept the change like that.

Reply:
> Unfortunately, the parameter order is now confusing. It should be at least: `colorXCenterLine`, `colorZCenterLine`, `colorGrid`. Or even better: `colorGrid`, `colorXCenterLine`, `colorZCenterLine`. However, this will break backwards compatibility.

I had noticed it too. Let me know if you need help with any code changes.